src/cephadm/cephadm

```diff
@@ -6710,21 +6710,31 @@ class HostFacts():
         # type: () -> Dict[str, str]
         """Determine the security features enabled in the kernel - SELinux, AppArmor"""
         def _fetch_selinux() -> Dict[str, str]:
-            """Read the selinux config file to determine state"""
+            """Get the selinux status"""
             security = {}
             for selinux_path in HostFacts._selinux_path_list:
```

Comment:
The code grabbing the info from sestatus lgtm. If we aren't going to use the selinux config file anymore though we should probably remove this for loop on the selinux path list (and probably the _selinux_path_list attribute from HostFacts entirely since this is its only use).
```suggestion
```

Reply:
I agree and, since the purpose of this class is to get the host facts and not to change the config, I do not think it will be ever needed.

I have done the changes.